help.go

```diff
@@ -39,3 +42,33 @@ func (e *Executor) tasksWithDesc() (tasks []*taskfile.Task) {
 	sort.Slice(tasks, func(i, j int) bool { return tasks[i].Task < tasks[j].Task })
 	return
 }
+
+// PrintTaskNames prints only the task names in a taskfile.
+func (e *Executor) PrintTaskNames() error {
+	// if called from cmd/task.go, e.Taskfile has not yet been parsed
+	if nil == e.Taskfile {
```

Comment:
```suggestion
	if e.Taskfile == nil {
```

Reply:
I'm new to Go, but often see your change here as _the convention_ when comparing equality with literals and variables. 

I was taught (with idioms from certain other barbaric languages) to put literals/constants on the left-hand side, as to avoid accidental assignment (`=` vs `==`), which isn't an issue with Go (for a couple reasons). 

Anyway, is this just style/preference or is there a pragmatic difference here I should be aware of?